contracts/transfer/tests/transfer.rs

```diff
@@ -383,29 +383,32 @@ fn withdraw_from_obfuscated() {
 
     let change_derive_key = DeriveKey::new(false, &change_psk);
 
-    let wfo_input = WfoCommitment::new(
-        message_value,
-        message_blinder,
-        *message.value_commitment(),
-    );
-
-    let wfo_change = WfoChange::new(
-        change,
-        change_value,
-        change_blinder,
-        change_r,
-        change_pk_r,
-        change_derive_key,
-    );
-
-    let wfo_output = WfoCommitment::new(
-        withdraw_value,
-        withdraw_blinder,
-        *withdraw.value_commitment(),
-    );
-
-    let wfo_circuit =
-        WithdrawFromObfuscatedCircuit::new(wfo_input, wfo_change, wfo_output);
+    let wfo_input = WfoCommitment {
```

Comment:
Why you didn't use a similar approach that you used for the obfuscated? Try to be consistent.